benchmarker.py

```diff
@@ -210,8 +211,9 @@ def benchmark_folder(
 
     print("==================")
     print("creating graphs")
-    outfile = [outheader]
-    errfile = [outheader[:]]
+    processed_table: list[list[int]] = []
```

Comment:
```suggestion
    processed_table: list[list[float]] = []
```
a great example of the uselessness of mypy and annotations

Reply:
huh?

Reply:
> huh?

I would say it's a problem of Any. where a function with Any as returntype leads to mypy not doing anything with the result.

Reply:
fixed in db25f66